src/directives/message_quote.ts

```diff
@@ -36,7 +36,7 @@ export default [
             template: `
                 <div class="message-quote-content" ng-style="{'border-color': ctrl.contact.color}">
                     <span class="message-name" ng-style="{'color': ctrl.contact.color}">{{ ctrl.contact.displayName }}</span>
-                    <span class="message-quote" ng-bind-html="ctrl.text | escapeHtml | nlToBr | emojify | linkify | markify"></span>
+                    <span class="message-quote" ng-bind-html="ctrl.text | escapeHtml | markify | nlToBr | emojify | linkify"></span>
```

Comment:
Can you move the `nlToBr` filter to the end, so that the order is the same as in `message_text.ts`?